config.md

```diff
@@ -128,27 +131,25 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-
-For Linux-based systems the process structure supports the following process specific fields:
-
-* **`capabilities`** (array of strings, OPTIONAL) capabilities is an array that specifies Linux capabilities that can be provided to the process inside the container.
-  Valid values are the strings for capabilities defined in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html).
+* **`capabilities`** (array of strings, OPTIONAL) is an array that specifies the set of capabilities of the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [CAPABILITIES(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html) man page.
```

Comment:
> As far as I know, there is no requirement for a minimum set of platform-specific configuration fields to be supported.

I agree that the previous language made this unclear, and that we want to add clarity moving forward.

> For this specific field, I've changed the statement that valid values for all platforms are documented in a Linux man page…

The old wording makes sense in master, where we [restrict `process.capabilities` to Linux][1].  A narrow expansion (to support Solaris as well) would be:

> On Linux, valid values are the strings for capabilities defined in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html).
> On Solaris, valid values are the strings for capabilities defined in [wherever Solaris documents these].

By using “For example”, it goes from being ambiguous about runtime support to reading more like “look in capabilities(7) for inspiration, but the runtime may not support all the capabilities documented there”, which is less ambiguous (in the direction of allowing a compliant no-op runtime).

And by extending the field to Windows (by lifting the Linux-only restriction), we open up a new can of worms (having to figure out what the Windows analog to capabilities is, assuming there is one with [capabilities not even being in POSIX][2]).

>  I would be fine with a discussion around whether or not a minimum amount of functionality must be implemented, but I don't see how that has bearing on this set of changes

I think the “For example” wording moves us further from where I'd like to be, with a (possibly platform-dependent) defined set of valid values which runtimes MUST support.  But the wording is ambiguous in master, and still ambiguous in ab7cca36 (is “For example” saying “these values may not work on compliant non-Linux runtimes but MUST work on compliant Linux runtimes” or  “these values may not work on compliant runtimes, even if they are Linux runtimes”?).  So I'm ok punting the discussion on what runtimes MUST support for now, landing whatever folks want in the meantime, and coming back around to “what MUST runtimes support?” in a different issue/PR (hopefully to be resolved before 1.0).

[1]: https://github.com/opencontainers/runtime-spec/blame/7278567eb2fbf95e96cc5ad8b3c22dc8fc81ecba/config.md#L132
[2]: http://wt.tuxomania.net/publications/posix.1e/